server/channels/api4/emoji.go

```diff
@@ -232,6 +239,36 @@ func getEmojiByName(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getEmojisByNames(c *Context, w http.ResponseWriter, r *http.Request) {
+	names := model.ArrayFromJSON(r.Body)
+	if len(names) == 0 {
+		c.SetInvalidParam("names")
+		return
+	}
+
+	if !*c.App.Config().ServiceSettings.EnableCustomEmoji {
+		c.Err = model.NewAppError("getEmojisByNames", "api.emoji.disabled.app_error", nil, "", http.StatusNotImplemented)
+		return
+	}
+
+	if len(names) > GetEmojisByNamesMax {
```

Comment:
I don't know if we usually define a max value for this type of API, but it seemed like a good idea to. I set it to 200 based on the max page size for any of those APIs

Reply:
i am wondering whats the scenario when the length is greater than max names

Reply:
It'd be hard to set up, but I think you could set it up by having DMs for 200+ users that all have different custom status emojis set